tests/utils.js

```diff
@@ -51,7 +51,110 @@ const basicTwoCrossTwoNodeWithFourChildren = {
   ]
 };
 
+const basicTestgridTemplateRowsColumnsNotProvided2 = {
+  style: {
+    display: 'grid',
+    width: 400,
+    height: 400,
+    justifyItems: 'center'
+  },
+  children: [
+    {
+      style: {
+        width: 100,
+        height: '100',
+        gridRowStart: '1',
+        gridRowEnd: '2',
+        gridColumnStart: '1',
+        gridColumnEnd: '2'
+      }
+    },
+    {
+      style: {
+        width: 100,
+        height: '100',
+        gridRowStart: 'one',
+        gridRowEnd: 'two',
+        gridColumnStart: 'two',
+        gridColumnEnd: '3'
+      }
+    },
+    {
+      style: {
+        width: 100,
+        height: '100',
+        gridRowStart: '2',
+        gridRowEnd: '3',
+        gridColumnStart: '1',
+        gridColumnEnd: '2'
+      }
+    },
+    {
+      style: {
+        width: 100,
+        height: '100',
+        gridRowStart: 'two',
+        gridRowEnd: 'three',
+        gridColumnStart: 'two',
+        gridColumnEnd: 'three'
+      }
+    }
+  ]
+};
+
+const basicTestgridTemplateRowsColumnsNotProvided = {
```

Comment:
@siddharthapaul001 Please try and provide a more relevant name to the variable.

Reply:
Yes I renamed it in latest commit.